genome_browser/_units.py

```diff
@@ -32,17 +32,16 @@ def __init__(
         start,
         end,
         strand='.',
-        name=None,
+        name='',
         **metadata
     ):
         assert isinstance(start, int), 'Loci must be integers'
         assert isinstance(end, int), 'Loci must be integers'
-        assert end - start > 0, 'Exclusive end must be greater than start'
```

Comment:
Used min/max below instead.
A friendlier API, and I can't see a situation where it would go wrong.